client/src/models/index.js

```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var path = require('path');
+var router = new express.Router();
+
+
+
+router.use('/bucketlist', require('./bucketlist'));
+
```

Comment:
Don't think this should say bucketlist? Also this file should be under controllers?

Reply:
I'm going to work on my proof reading

Reply:
what file are those in?

Reply:
<img width="233" alt="image" src="https://user-images.githubusercontent.com/18101421/34060493-e9884308-e1db-11e7-82b1-a569e91dff4e.png">

Reply:
the file shouldn't have been there, I deleted it.